test/adaptors/postgres-articles.test.ts

```diff
@@ -0,0 +1,147 @@
+import { blankNode } from '@rdfjs/data-model';
+import all from 'it-all';
+import { JsonLdObj } from 'jsonld/jsonld-spec';
+import { BlankNode } from 'rdf-js';
+import pgPromise, { IBaseProtocol, IMain } from 'pg-promise';
+import pgSetup from '@databases/pg-test/jest/globalSetup';
+import pgTeardown from '@databases/pg-test/jest/globalTeardown';
+import getDatabase from '@databases/pg-test';
+import PostgresArticles from '../../src/adaptors/postgres-articles';
+import ArticleNotFound from '../../src/errors/article-not-found';
+import NotAnArticle from '../../src/errors/not-an-article';
+import { schema } from '../../src/namespaces';
+import createArticle from '../create-article';
+import CreateDb from '../../src/tools/create-db';
+
+let database: IBaseProtocol<IMain>;
+
+beforeAll(async (): Promise<void> => {
+  jest.setTimeout(10000);
+  database = await getDatabase().then((db) => pgPromise()(db.databaseURL));
```

Comment:
I can see https://github.com/rrwen/pg-testdb/blob/master/index.js depends on https://github.com/olalonde/pgtools which has a JavaScript implementation of `createdb` and `dropdb` to manage logical database inside a Postgres server. This is within the realm of possibility, if performance allows (as the minimal teardown for tests or even test runs would be to clean the table as this `beforeEach` does, rather than using completely new databases).

I'm looking for signs of a new Postgres server being run, which would be out of place for application/test code to do; but I'm still digging.

Reply:
Ah got the wrong one: https://github.com/ForbesLindesay/atdatabases/tree/master/packages/pg-test has a dependency on `@databases/with-container` which contains the [`docker` commands](https://github.com/ForbesLindesay/atdatabases/blob/master/packages/with-container/src/index.ts).

Reply:
We are no longer using this approach.